src/admin/WL_Metabox/WL_Metabox_Field_address.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ * Field to manage the address. The pattern followed is to simply build an array of subfields using the base WL_Metabox_field class,
+ * and act as a proxy between WL_Metabox and them.
+ * 
+ * @since 3.2.0
+ */
+class WL_Metabox_Field_address extends WL_Metabox_Field {
+
+	/**
+	 * Sub-fields contained in the Field.
+	 *
+	 * @since 3.2.0
+	 * @access private
+	 * @var $subfields Array of WL_Metabox_Field objects, each dealing with a part of the http://schema.org/PostalAddress structure.
+	 */
+	private $subfields;
+	
+	/**
+	 * Constructor.
+	 * 
+	 * @param array $args Set of fields containing info to build the subfields.
+	 */
+	public function __construct( $args ) {
+		
+		// leverage the WL_Metabox_Field class to build the subfields
+		$this->subfields = array();
+		foreach ( reset($args) as $key => $subfield ) {
```

Comment:
I use there `reset` to take the first element of the array which in turn contains an array (the looped one).